gimie/project.py

```diff
@@ -20,7 +20,7 @@
 from typing import Iterable, List, Optional, Union
 from gimie.graph.operations import combine_graphs
 from gimie.utils import validate_url
-from gimie.sources.helpers import (
+from gimie.sources.helpers_project import (
```

Comment:
I _feel_ like there is a distinction to be made between "helpers" that simplify working with the sources (e.g. from `Project`), and those used by the sources themselves to abstract common elements (meant for internal use).

What are your thoughts on putting `helpers_project` functions in `gimie/sources/__init__.py`, and scoping all "internal helpers" together in `gimie.sources.common` so they don't stand at the same level as sources?

This way, individual sources would do:

```py
from gimie.source.common.queries import send_rest_query
```
and external libraries (or `Project`) would do
```py
from gimie.sources import is_valid_source
```

Reply:
I like the idea of `from gimie.source.common.queries import send_rest_query` 

But for the `helpers_project` in the `__init__` file, I'm less convinced. Could we also have them at the same level as `project.py` ? i.e. `from gimie.helpers_project import is_valid_source` ?

Reply:
I moved the project specific functions from `helpers_project.py` inside the `project.py` script